Optimobile/AppDelegate.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  AppDelegate.swift
+//  Optimobile
+//
+//  Created by Barak Ben Hur on 04/04/2022.
+//
+
+import UIKit
+import KumulosSDK
+
+@main
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        let builder = KSConfigBuilder(apiKey: "2b16f7c7-6f0e-46c1-979f-1c13b5fd8a0f", secretKey: "Xh62vshcS15xJCjSHiQ2dcPvfXHVirKdT+Vg").enableInAppMessaging(inAppConsentStrategy: InAppConsentStrategy.AutoEnroll).setPushOpenedHandler(pushOpenedHandlerBlock: { (notification : KSPushNotification) -> Void in
```

Comment:
Is there a test app as part of the SDK bundle? Im not used to seeing credentials checked in.